package.json

```diff
@@ -87,6 +87,7 @@
         "@babel/preset-typescript": "^7.16.5",
         "@deriv/api-types": "1.0.48",
         "@testing-library/jest-dom": "^5.12.0",
+        "@types/react-transition-group": "^4.4.4",
```

Comment:
Question: Why do we need to include this?

Reply:
when changing the jsx to tsx the `import { CSSTransition } from 'react-transition-group';` get an typescript error, we need to add this package to fix that.

Reply:
Got it thanks 🙏

Reply:
:pray: